Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM        quay.io/prometheus/busybox:latest
-MAINTAINER  Alexey Palazhchenko <alexey.palazhchenko@percona.com>
+FROM golang:1.11 as builder
 
-COPY mongodb_exporter /bin/mongodb_exporter
+MAINTAINER Ekambaram Pasham <ekambaram_pasham@infosys.com>
```

Comment:
Do you really intend to maintain it?

Reply:
i can maintain the build and deployment configuration